src/gui/tray/activitydata.cpp

```diff
@@ -127,7 +127,7 @@ OCC::Activity Activity::fromActivityJson(const QJsonObject json, const AccountPt
         activity._previews.append(data);
     }
 
-    if(!previewsData.isEmpty()) {
+    if(previewsData.isEmpty()) {
```

Comment:
could you put this into another PR that seems unrelated to dark mode fixes

Reply:
It's not unrelated, it affects which icons are used in the activtylistmodel for the dark/light icons

Reply:
Removed this change